packages/zoe/test/unitTests/zcf/zcfTesterContract.js

```diff
@@ -1,6 +1,7 @@
 // @ts-check
 
 import '../../../exported.js';
+import { Far } from '@agoric/marshal';
```

Comment:
FWIW, as of #3833 I think the preferred idiom is:

```suggestion
import { Far } from '@agoric/far';
```

Reply:
I'll get this one, but there are too many others to think about.